### DISCUSSION TITLE
Select rtlsdr versoin by device number

### DISCUSSION THREAD

**Discussion by gariac** [rsadsb/dump1090_rs#131] [Q&A]
It would be useful if the rtlsdr version could select by device number.

**Comment by wcampbell0x2a** [comment]
You should be able to use `--driver-extra` like https://github.com/rsadsb/dump1090_rs/issues/102

**Comment by gariac** [comment]
Like they say on reddit, today I learned there are commands for soapysdr:

https://github.com/charlie-foxtrot/RTLSDR-Airband/wiki/Configuring-SoapySDR-devices

This will give me some things to hack. I have one of those Chinese MSI devices I could never get running. Thanks.
